content/v0.75.2.md

```diff
@@ -0,0 +1,129 @@
++++
+title = "0.75.2 Maintenance Release Notes"
+date = 2020-10-25
++++
+
+Installation instructions:
+
+- [Linux](/downloads/linux/) (or [Steam on Linux](/downloads/linux#steam))
+- [Windows](/downloads/windows/)
+- [macOS](/downloads/macos/)
+
+## Table of Contents
+
+- [1. Summary](#1-summary)
+- [2. New Features in Detail](#2-new-features-in-detail)
+  - [2.1. Select Monochrome Palette Colour](#2-1-select-monochrome-palette-colour)
+  - [2.2. Raw Mouse Input](#2-2-raw-mouse-input)
+- [3. Bugfixes](#3-bugfixes)
+- [4. Notes for Packagers](#4-notes-for-packagers)
+- [5. Contributors](#4-contributors)
+- [6. Changelog](#5-changelog)
+
+## 1. Summary
+
+This is a maintenance release from 0.75.x stable branch - it includes a few
+changes and bugfixes backported from upcoming 0.76.x version.
```

Comment:
Consistency suggestion:

... release from the **_stable_** 0.75.x <something>
... backported from the **_upcoming_** 0.76.x <something>

Where <something> is maybe _release_ or _branch_ or .. other name? (I'm not set on any.. just that we use the same terminology to refer in both cases).

Reply:
Hmm, I was trying to be precise, as we don't have 0.76.x release branch yet. Maybe I'll just drop the mention of backport - it does not add anything new to the release notes.

Reply:
> we don't have 0.76.x release branch yet

That's ok, because _upcoming_ implies that whatever you want to call it, that that's still to-come.

It's fine to say:

from the stable 0.75.x release.. back-ported from the upcoming 0.76.x release.